_posts/2014-04-05-overview.md

```diff
@@ -5,30 +5,37 @@ permalink: overview
 github: "https://github.com/ember-cli/ember-cli.github.io/blob/master/_posts/2014-04-05-overview.md"
 ---
 
-Ember CLI is the [Ember.js](http://emberjs.com) command line utility that
-provides a fast [Broccoli](https://github.com/broccolijs/broccoli#broccoli)-powered
-asset pipeline, a strong conventional project structure, and a powerful [addon
-system](http://www.ember-cli.com/extending/#developing-addons-and-blueprints)
-for extension. [Get started.](/user-guide/#getting-started)
+Ember CLI is the official command line utility used to create, build, serve, 
+and test [Ember.js](http://emberjs.com) apps and addons.
+
+It provides a consistent project structure, a powerful [addon
+system](http://www.ember-cli.com/extending/#developing-addons-and-blueprints), and
+a fast [Broccoli](https://github.com/broccolijs/broccoli#broccoli)-powered
+asset pipeline.
```

Comment:
I'd suggest the phrase "build pipeline" instead of "asset pipeline." In my experience, "asset pipeline" is a phrase *primarily* familiar to people from the Rails world.